packages/adminrouter/extra/src/includes/http/common-windows.conf

```
@@ -0,0 +1,135 @@
+
+#vhost_traffic_status_filter_max_node 2700;
+
+# This implies in the worst-case metrics for 2700 / 3 (for status, URI, client)
+# = 900 requests can be safely stored at any given time. Given a Telegraf
+# scrape interval of 10 seconds this means 90 requests/s is a safe rate at
+# which we expect no metrics to be missed despite the worst-case (status, URI,
+# client) tuple distribution. In MWT #11 we did not see 100 requests/s being
+# reached across all Master Admin Router instances so this should suffice.
+#
+# The aforementioned numbers were evaluated in the following experiment:
+#
+# 1. Call random 128 characters long random URLs on Admin Router in a loop.
+#
+# 2. Wait for the shared memory to fill up until the LRU mechanism activates.
+# This resulted in in 2701 nodes being used -> 9.2MB additional metric data.
+#
+# 3. Node internal measurements (Telegraf perspective):
+# - curl http://localhost/nginx/metric (Downloading 10.6MB of data takes < 1
+# second, nearly instant even.)
+#
+# 4. Node external measurements (Telegraf endpoint):
+# - curl http://172.17.0.2:61091/metrics takes < 3 seconds on Docker
+# (Downloading 53MB of metric data on an otherwise empty cluster)
+#
+# The scraping interval of Prometheus in the dcos-monitoring package currently
+# is 60 seconds and the timeout at 50 seconds. This means it would need to
+# scrape the Telegraf endpoint at roughly 1MB/s when the shared memory is
+# filled up in this experiment.
+#
+# Going forward we will implement a dedicated Admin Router Telegraf plugin
+# which vastly reduces the number of reported useless metrics.
+# From then on, only the node internal scraping time for Nginx metrics is
+# impacted by the settings above, which eases decision making on the
+# maintainers of this configuration. Until then, this the settings above are
+# believed to present a reasonable trade-off to accommodate metrics for
+# internal testing and not prevent Prometheus from falling behind on scraping
+# metrics from Telegraf running on each DC/OS node.
+
+client_max_body_size 1024M;
+
+# Define custom log format. Use the default 'combined'
+# format and append easily parsable timing information
+# for performance analysis, and other relevant debugging
+# information.
+log_format customformat '$remote_addr - $remote_user [$time_local] '
+    '"$request" $status $body_bytes_sent '
+    '"$http_referer" "$http_user_agent" '
+    'logtime_msec=$msec '
+    'upstream_addr=$upstream_addr '
+    'request_time=$request_time '
+    'upstream_response_time=$upstream_response_time '
+    'upstream_connect_time=$upstream_connect_time '
+    'upstream_header_time=$upstream_header_time';
+
+# The syslog facility here is set to daemon because
+# systemd SyslogFacility defaults to daemon and
+# therefore all other DC/OS services log to it.
+# https://jira.mesosphere.com/browse/DCOS-38622
+access_log C:/dcos/mesos-logs/dcos-adminrouter-access.log,facility=daemon customformat;
```

```suggestion
access_log C:/dcos/logs/dcos-adminrouter-agent.log,facility=daemon customformat;
```
For consistency with Linux agents, we should use similar naming.